autobot/telegram/parser.py

```diff
@@ -697,3 +697,437 @@ def _parse_sticker (self,key:str,val:dict) -> Sticker|StickerSet:
 
         return sticker_obj
 
+
+    def _parse_animation(self,key:str,val:dict) -> Animation:
+        animation_obj = Animation()
+        for k,v in val.items():
+            match k:
+                case "thumb":
+                    thumb = self._parse_photosize(k,v)
+                    setattr(animation_obj,k,thumb)
+                case _:
+                    setattr(animation_obj,k,v)
+        return animation_obj
+
+    
+    def _parse_audio(self,key:str,val:dict) -> Audio:
+        audio_obj = Audio()
+        for k,v in val.items():
+            match k:
+                case "performer"|"title":
+                    setattr(audio_obj,k,v)
+
+                case "file_size":
+                    setattr(audio_obj,k,int(v))
+                case _:
+                    setattr(audio_obj,k,v)
+        return audio_obj
+
+
+    def _parse_video(self,key:str,val:dict) -> Video:
+        video_obj = Video()
+        for k,v in val.items():
+            match k:
+                case "thumb":
+                    thumb = self._parse_photosize(k,v)
+                    setattr(video_obj,k,thumb)
+
+                case "file_name"|"mime_type":
+                    setattr(video_obj,k,v)
+
+                case "file_size":
+                    setattr(video_obj,k,int(v))
+                case _:
+                    setattr(video_obj,k,v)
```

Comment:
You can remove the first two case statements and only remain with
`case _:` so we should have
```python
def _parse_video (self,key:str,val:dict) -> Video:
    video_obj = Video()
    for k,v in val.items():
        match k:
            case "thumb":
                thumb = self._parse_photosize(k,v)
            case _:
                setattr(video_obj,k,v)
    return video_obj
```
also on line 739, we shouldn't manipulate results during parsing, such as `int(v)`

This method should be also include ability to parse the rest of the objects in the video module, such as `VideoChatEnded`,`VideoChatScheduled`,`VideoChatStarted` etc